protocol/src/main/kotlin/tbdex/sdk/protocol/models/Rfq.kt

```diff
@@ -46,13 +48,14 @@ class Rfq private constructor(
 
   private fun validatePaymentMethod(selectedMethod: SelectedPaymentMethod, offeringMethods: List<PaymentMethod>) {
     val matchedOfferingMethod = offeringMethods.first { it.kind == selectedMethod.kind }
-    if (matchedOfferingMethod.requiredPaymentDetails !== null) {
-      matchedOfferingMethod.requiredPaymentDetails.validate(selectedMethod.paymentDetails.toString())
+    matchedOfferingMethod.requiredPaymentDetails?.let {
+      val schema = matchedOfferingMethod.getRequiredPaymentDetailsSchema()
+      val jsonNodePaymentDetails = Json.jsonMapper.valueToTree<JsonNode>(selectedMethod.paymentDetails)
+      schema?.validate(jsonNodePaymentDetails)
     }
-
   }
 
-  private fun verifyClaims(requiredClaims: List<PresentationDefinitionV2>) {
+  private fun verifyClaims(requiredClaims: PresentationDefinitionV2) {
```

Comment:
if multiple VCs required, are we smushing into a single PD?

Reply:
This change is to match the json schema. Maybe @mistermoe knows?